packages/firestore/test/integration/api/database.test.ts

```diff
@@ -31,6 +31,7 @@ import {
   apiDescribe,
   withTestCollection,
   withTestDb,
+  withTestDbNoShutdown,
```

Comment:
leftover?

Reply:
removed in subsequent commit, another careless error.